frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -34,18 +35,25 @@ import {
   storageClassTooltip,
   defaultRequestSize,
 } from '../../../constants';
-import { OCSStorageClassDropdown } from '../storage-class-dropdown';
+import { filterSC, isArbiterSC, isValidTopology } from '../../../utils/install';
 import { PVsAvailableCapacity } from '../../ocs-install/pvs-available-capacity';
 import { pvResource, nodeResource } from '../../../resources';
 import { createDeviceSet, getDeviceSetCount } from '../../ocs-install/ocs-request-data';
 import { DeviceSet } from '../../../types';
 import './add-capacity-modal.scss';
-import { isArbiterSC, isValidTopology } from '../../../utils/install';
+
 import { checkArbiterCluster, checkFlexibleScaling } from '../../../utils/common';
 
 const queries = (() => Object.values(CAPACITY_INFO_QUERIES))();
 const parser = compose((val) => val?.[0]?.y, getInstantVectorStats);
 const getProvisionedCapacity = (value: number) => (value % 1 ? (value * 3).toFixed(2) : value * 3);
+const getSelectedKey = (
+  storageClass: StorageClassResourceKind,
+  deviceSets: DeviceSet[],
+): string => {
+  if (storageClass) return getName(storageClass);
+  return deviceSets?.[deviceSets.length - 1]?.dataPVCTemplate?.spec?.storageClassName;
```

Comment:
Please add a comment for this behaviour.

Reply:
ack...
but, there is one thing though:
If I am using same storageClass for adding capacity twice, then, that storageClass will not necessarily be always at  `deviceSets[deviceSets.length - 1]` index...
Example:
Initial deployment using `gp2`
Expand for the first time using `gp2-csi`
Expand for the second time using `gp2`
In the above case `gp2` will still be at 0th index of the deviceSets and when we will Expand for the third time `gp2-csi` will be the pre-selected value as it is present at (length-1)th index...

Reply:
Correct, we can only show the last device set index or first device set index.
In practice, the use cases have been around using another storage class when a user wants to use new storage devices. 
But the possibilities and use-cases are beyond our control and let's keep it close to generic so let's revert this to the first storage device set only and add a proper comment.
That is, always show the installation storage class.